aten/src/TH/generic/THBlas.cpp

```diff
@@ -380,6 +383,39 @@ void THBlas_(gemm)(
     return;
   }
 #endif
+
+#ifdef USE_FBGEMM
+#if defined(TH_REAL_IS_LONG)
+  if (alpha == 1 && (beta == 0 || beta == 1)) {
```

Comment:
it looks like beta==1 by default, do we have any unit tests that cover `beta != 1`? I'm not familiar with fbgemm so not sure what happens when `accumulate==false`.

Reply:
If accumulate== false, we compute C = A * B;
otherwise (accumulate==true), we compute C += A * B.

I think these two cases are mainly used by the Linear module in PyTorch: we rarely use other `beta`.

Re "it looks like beta==1 by default, do we have any unit tests that cover beta != 1":
Right: in "addmm", we need to have beta == 1 as we need to accumulate the the result matrix C. In FBGEMM, we have the test coverage for both accumulate == false/true (beta == 0/1).

Reply:
> we rarely use other beta.

That's what I figured, and even in existing code it's always possible we made a mistake somewhere in how the beta is passed through all these functions. That explanation makes sense though and probably sufficient for this specific change.